### PR TITLE
Adding x86 definition

### PR DESCRIPTION
**Pull request by siddhpant**
This PR defines the x86 assembly language which was earlier not defined and supported by coAST.

This definition is a step closer for adding x86 to coala.

### CODE REVIEW

File: data/Delimiter/semicolon.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+type: comment
+start: ";"
```
Comment (ksdme) [inline data/Delimiter/semicolon.yaml]: Trailing new line.

**Comment by ksdme** [comment]
Squash your commits please.

File: data/Delimiter/semicolon.yaml (hunk 1 above)
Comment (siddhpant) [reply data/Delimiter/semicolon.yaml]: You commented on the other file :)

Comment (siddhpant) [reply data/Delimiter/semicolon.yaml]: Fixed there

**Comment by siddhpant** [comment]
Done.

**Comment by jayvdb** [comment]
This has too many values in `keywords`.  It breaks statik.

I think the limit is aboutt  1000 keywords ; maybe 1024.  Maybe a bug in statik that we can fix, but ... this list contains upper and lower case.  Lets not explicitly list both.  case sensitivity is a separate property of languages, and/or of specific keywords.  We can model that later.

**Comment by siddhpant** [comment]
ohk

**Comment by siddhpant** [comment]
Removed uppercase keywords.

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by jayvdb** [comment]
Actually it might be a bug in a recent merge to statik. I'll take a look..

**Comment by jayvdb** [comment]
Yup, https://github.com/thanethomson/statik/pull/93 is the problem, so we'll fix that shortly

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by jayvdb** [comment]
Gah.  Please squash your commits.  The main repo's permanent history doesnt need stuff added and then the same stuff removed immediately afterwards.

**Comment by siddhpant** [comment]
Okay!

I didn't squash in case the other case was needed.

Squashed. Should be good to go.

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/commit/01100420e6129171bbbaed99deb7e9cb3ac0040b

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**